rpc_storage_handler.go

```diff
@@ -422,8 +422,30 @@ func (r *RPCStorageHandler) GetKey(keyName string) (string, error) {
 }
 
 func (r *RPCStorageHandler) GetRawKey(keyName string) (string, error) {
```

Comment:
@joshblakeley do you think we can now rewrite `GetKey` function to use `GetRawKey` internally?